src/coreclr/scripts/superpmi_asmdiffs_setup.py

```diff
@@ -167,18 +171,22 @@ def main(main_args):
         os.makedirs(base_jit_directory)
 
     print("Fetching history of `main` branch so we can find the baseline JIT")
-    run_command(["git", "fetch", "origin", "main"], source_directory, _exit_on_fail=True)
+    run_command(["git", "fetch", "--depth=500", "origin", "main"], source_directory, _exit_on_fail=True)
 
     # Note: we only support downloading Windows versions of the JIT currently. To support downloading
     # non-Windows JITs on a Windows machine, pass `-host_os <os>` to jitrollingbuild.py.
-    print("Running jitrollingbuild.py download to get baseline")
+    print("Running jitrollingbuild.py download to get baseline JIT")
+    jit_rolling_build_script = os.path.join(superpmi_scripts_directory, "jitrollingbuild.py")
     _, _, return_code = run_command([
         python_path,
-        os.path.join(superpmi_scripts_directory, "jitrollingbuild.py"),
+        jit_rolling_build_script,
         "download",
         "-arch", arch,
         "-target_dir", base_jit_directory],
         source_directory)
+    if return_code != 0:
```

Comment:
You need to make sure that you return the actual `error_code` from `jitrollingbuild.py` file. Currently it always return `success`.

Reply:
It turns out that doesn't matter: jitrollingbuild.py error model currently is to throw an exception, and that leads to a process error code of `1`. I started changing jitrollingbuild.py, then realized it wasn't necessary.

Reply:
Got it. Can you remove the unused `success = True` from the `jitrollingbuild.py` then?

Reply:
Done